docs/src/dev_guide/tests.md

```diff
@@ -1,28 +1,31 @@
 # Running Tests
 
 ## Standard test execution
+
 Unit tests can be executed in the REPL by executing the following:
 
 ```julia
 julia> ] test
 ```
 
 ## Interactive test execution
+
 While developing code and tests it can be convenient to run a subset of tests.
 You can do this with a combination of TestEnv.jl and ReTest.jl.
 
 **Note**: Per recommendations from the developers of TestEnv.jl, install the package
 in your global julia environment. Do the same for Revise.jl.
 
 ```
-$ julia
+\$ julia
```

Comment:
What are the backslashes for?

Reply:
The documenter throws an error if there are not escaped $ symbols

Reply:
I reverted the change